recipes/from_package.rb

```diff
@@ -34,9 +34,6 @@
     package "(#{type}) #{pkg}" do
       package_name pkg
       only_if { Dovecot::Conf.require?(type, node['dovecot']) }
-      if type == 'core' || node['dovecot']['ohai_plugin']['build-options']
-        notifies :reload, 'ohai[reload_dovecot]', :immediately
-      end
```

Comment:
It is done by ohai_plugin
here: https://github.com/chef-cookbooks/ohai/blob/master/resources/plugin.rb#L83
here: https://github.com/chef-cookbooks/ohai/blob/master/resources/plugin.rb#L90